tools/sof-kernel-log-check.sh

```diff
@@ -104,9 +99,14 @@ case "$platform" in
     # Audio PCI ID on CML Mantis is [8086:9dc8], which is defined as CNL in linux kernel.
     # https://github.com/thesofproject/linux/blob/topic/sof-dev/sound/soc/sof/sof-pci-dev.c
     icl|cml|cnl)
-        # On CML_RVP_SDW, suspend-resume test case failed due to "mei_me 0000:00:16.4: hw_reset failed ret = -62".
+        # On CML_RVP_SDW, suspend-resume test case failed due to "mei_me 0000:00:16.4: hw_reset failed ret = -62" or with "hw_start" with same error code.
         # https://github.com/thesofproject/sof-test/issues/389
         ignore_str="$ignore_str"'|mei_me 0000:00:16\..: hw_reset failed ret = -62'
+        ignore_str="$ignore_str"'|mei_me 0000:00:16\..: hw_start failed ret = -62'
```

Comment:
This scripts uses a -E(xtended) expression so `(reset|start)` should work too if you prefer.
```shell
echo hw_start | grep -E 'hw_(reset|start)'
```

Reply:
@marc-hb I contemplated using that, but I guess there is value in keeping the text strings as verbatim copies of dmesg output, so when you search for errors you found in dmesg, you'll catch them more easily.

Reply:
Good point... deserving a comment so no one comes in the future and "refines" this line with my suggestion?

Reply:
@marc-hb That applies to the whole file really, right?

Reply:
@kv2019i  scores again. I will add this at the top.

Reply:
> but I guess there is value in keeping the text strings as verbatim copies of dmesg output, so when you search for errors you found in dmesg, you'll catch them more easily.

Submitted (and A LOT more) as a guideline in #470